frontend/public/components/namespace.jsx

```diff
@@ -136,14 +136,36 @@ const ProjectTableHeader = () => {
 };
 ProjectTableHeader.displayName = 'ProjectTableHeader';
 
-const ProjectTableRow = ({obj: project, index, key, style}) => {
+const projectLink = connect(
+  null,
+  { setActiveNamespace: UIActions.setActiveNamespace },
+)(({ project, setActiveNamespace }) => (
+  <span className="co-resource-item co-resource-item--truncate">
+    <ResourceIcon kind="Project" />
+    <button
```

Comment:
Should add e `title` to be consistent.

Reply:
ResourceIcon doesn't have title, only ResourceLink component has the title property

Reply:
But this button doesn't have the title.

Reply:
added title prop to the button